NitroxModel-Subnautica/DataStructures/DataExtensions.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using NitroxModel.DataStructures.GameLogic;
+using UnityEngine;
+
+namespace NitroxModel_Subnautica.DataStructures
+{
+    public static class DataExtensions
+    {
+        public static NitroxVector3 ToDto(this Vector3 v)
+        {
+            return new NitroxVector3(v.x, v.y, v.z);
+        }
+
```

Comment:
I'd like to keep this ambiguous and not Nitrox specific in case we want to extend this to external services (outside of Nitrox).

Reply:
Ok, after your message in discord I know the meaning and I am fine with it :)

Reply:
What does "Dto" mean? 
Maybe better use .ToUnityVector() and .ToNitroxVector() ? Then it should be clear enough what this method concrete does and not be too ambigous to confound with other "ToNitrox" conversions.

Reply:
@acidscorch `ToDto` means `To data transfer object` which is often used for such "data conversation". `ToNitroxVector` is (imo) too complicated because you already use it on a type like `NitroxVector`. I would just wait for @Sunrunner37 s opinion.

Reply:
@Jannify Ah yes, i understand. (Sorry i'm not very familiar with extension methods.) 
This Method is for converting a UnityVector to a NitroxVector, correct? And you already have a .ToUnity() Method. Maybe then use .FromUnity() for this?